go/ct/common/revisions_test.go

```diff
@@ -85,3 +86,27 @@ func TestRevision_GetRevisionForBlock(t *testing.T) {
 	}
 
 }
+
+func TestRevisions_GetForkTime(t *testing.T) {
+	tests := map[tosca.Revision]struct {
+		revision tosca.Revision
+		forkTime uint64
+	}{
+		tosca.R07_Istanbul:      {tosca.R07_Istanbul, 0},
+		tosca.R09_Berlin:        {tosca.R09_Berlin, 1000},
+		tosca.R10_London:        {tosca.R10_London, 2000},
+		tosca.R11_Paris:         {tosca.R11_Paris, 3000},
+		tosca.R12_Shanghai:      {tosca.R12_Shanghai, 4000},
+		tosca.R13_Cancun:        {tosca.R13_Cancun, 5000},
+		R99_UnknownNextRevision: {R99_UnknownNextRevision, 6000},
+	}
+
+	for name, test := range tests {
+		t.Run(name.String(), func(t *testing.T) {
+			got := GetForkTime(test.revision)
```

Comment:
Since we are already here, why is `GetForkTime` not returning a `time.Time`? Fixing this may be a small follow-up cleanup.

Reply:
I think uint64 was used because in the reference implementation, uint64 is also used. Maybe it could be turned into `time.Time`, but that would be a separate PR